### PR TITLE
[proton-bridge] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
[proton-bridge](https://github.com/ProtonMail/proton-bridge) use by [protonMail](https://proton.me/mail) e-mail clients.

@LBeernaertProton Ping, Dose this look good?

### CODE REVIEW

**Comment by Alan32Liu** [comment]
> [proton-bridge](https://github.com/ProtonMail/proton-bridge) use by [protonMail](https://proton.me/mail) e-mail clients.

Would you please elaborate a bit more about: 1) Why it is critical? 2) Who are the main users?

**Comment by 0x34d** [comment]
> > [proton-bridge](https://github.com/ProtonMail/proton-bridge) use by [protonMail](https://proton.me/mail) e-mail clients.
> 
> Would you please elaborate a bit more about: 
> 1) Why it is critical?

Well, `proton-bridge` is used by Email clients to sync with `ProtonMail server`.

> 2) Who are the main users?

 Not, everyone. But if you want to use an email client for protonMail this is very important.

@LBeernaertProton is this information correct?

**Comment by LBeernaertProton** [comment]
Proton Bridge is a "bridge" between the proton servers and the user's IMAP client of choice. Due to the nature of Proton's ecosystem, the bridge runs an IMAP server on the user's machine where it decrypts and transforms the user's proton email into a format understood by the IMAP clients.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: projects/proton-bridge/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+RUN git clone --depth 1 https://github.com/ProtonMail/proton-bridge.git
+RUN go install github.com/AdamKorcz/go-118-fuzz-build@latest
```
Comment (jonathanmetzman) [inline projects/proton-bridge/Dockerfile:19]: Is this part still needed? Why?

Comment (0x34d) [reply projects/proton-bridge/Dockerfile:19]: > go install github.com/AdamKorcz/go-118-fuzz-build@latest

This one, I don't know. The last time I tried to change it build failed.

**Pull request merged**